wirecard/oxid-ee
# WDPD-172: Frontend payment method for payolution

### This PR

* Adds Input fields for date of birth and phone
* Adds select field for saving input data
* Adds mechanism to show/hide payment methods(e.g. hide guaranteed invoice if user < 18)

### Notes

* if basket is empty it is not possible to get to the payment method select step in oxid
* Guaranteed Invoice by Payolution payment method is only testable with country set to Austria, to be able to use Austria add it in backend (Master Settings->Countries)

### How to Test

* Check if input fields are correctly shown/hidden to/from user
* Check if save selection is correctly shown/hidden to/from user
* Check if saving/ not saving works correctly
* Check if showing/hiding of payment method works correctly
* Check if phone input field is only shown if Netherlands is chosen as country and not already set
* Test with logged in and guest user
* Test if Ratepay still works as expected

### Jira Links
https://jira.parkside.at/browse/WDPD-172
